server/src/util/event-email.ts

```diff
@@ -263,6 +290,17 @@ export const eventAttendanceConfirmation = ({
     end: event.ends_at,
     description: event.description,
   };
+
+  // Following the same format from eventInviteEmail,
+  // grabbing physical location and streaming data
+  // in the same manner
+  const physicalLocation = isPhysical(event.venue_type)
+    ? `\n${physicalLocationShortText(event.venue?.name)}`
+    : '';
+  const streamingData = isOnline(event.venue_type)
+    ? `\n${streamingURLText(event.streaming_url)}`
+    : '';
```

Comment:
These are used in few places, perhaps it's time to add helper function for each of them.

Reply:
I made some helper functions called getPhysicalLocation and getStreamingData near the top. I hope this is what you meant?